incubator/jaeger/templates/agent-svc.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: "{{ template "jaeger.fullname" . }}-agent"
+  labels:
+    app: "{{ template "jaeger" . }}"
+    jaeger-infra: {{ .Values.service.agent.name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    component: "{{ template "jaeger.fullname" . }}-agent"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+{{- if .Values.service.query.annotations }}
+  annotations:
+{{ toYaml .Values.service.query.annotations | indent 6 }}
+{{- end }}
+spec:
+  ports:
+  - name: agent-zipkin-thrift
+    port: {{ .Values.service.agent.zipkinThriftPort }}
+    protocol: UDP
+    targetPort: {{ .Values.service.agent.zipkinThriftPort }}
+  - name: agent-compact
+    port: {{ .Values.service.agent.compactPort }}
+    protocol: UDP
+    targetPort: {{ .Values.service.agent.compactPort }}
+  - name: agent-binary
+    port: {{ .Values.service.agent.binaryPort }}
+    protocol: UDP
+    targetPort: {{ .Values.service.agent.binaryPort }}
+  clusterIP: None
+  selector:
```

Comment:
Selector should have `app`, `component`, and `release` labels.

Reply:
Add app, component, and release to selector.